microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-   ```javascript
-   const express = require('express');
-   const app = express();
-   const fs = require('fs');
-   const compression = require('compression');
-   app.use(compression());
-   app.use(express.static('build'));
-   const listener = app.listen(process.env.PORT || 1234, function () {
-      console.log(`Listening on port ${listener.address().port}`);
-   });
-   ```
+*  Automatically resize images during your build process.
 
-   > [!NOTE]
-   > Usually, you have to install the `compression` package via something like `npm i -S compression`, but this has already been done for you.
+*  Create multiple sizes of each image during the build process and then use `srcset` in your code. At runtime, the browser takes care of choosing which size is best for the device. To learn more, see [Responsive images](https://web.dev/responsive-images/) at web.dev.
 
-1. Wait for Glitch to deploy the new build of the site.  The fancy animation next to **Tools** means that the site is getting rebuilt and redeployed.  The change is ready when the animation next to **Tools** goes away.  Click **Show** and then select **In a New Window** again.
+*  Use an image CDN that lets you dynamically resize an image when you request it.
 
-   <!--
-   ![The animation that indicates that the site is getting built](../media/speed-glitch-tony-remix-server-js-edited.msft.png)
-   -->
+*  At least, optimize each image.  can often create huge savings.
 
-Use the workflows that you learned earlier to manually check that the compression is working:
+Optimization is when you run an image through a special program that reduces the size of the image file.  For more tips, see [Essential Image Optimization](https://images.guide).
 
-1. Go back to the demo tab and refresh the page.  The **Size** column should now show 2 different values for text resources like `bundle.js`.  In the figure after the following, the top value of `256 KB` for `bundle.js` is the size of the file that was sent over the network, and the bottom value of `1.2 MB` is the uncompressed file size.
+#### Reduce unused JavaScript
 
-   ![The Size column now shows two different values for text resources](../media/speed-glitch-tony-remix-network-main.msft.png)
+Your latest **Lighthouse** report says that the webpage contains unused JavaScript code and that loading this code only when required would decrease the amount of data transferred when the page loads.
 
-1. The **Response Headers** section for `bundle.js` now includes a `content-encoding: gzip` header:
+Click **Reduce unused JavaScript** to reveal the JavaScript files that contain the most unused code:
 
-   ![The Response Headers section now contains a 'content-encoding' header](../media/speed-glitch-tony-remix-network-bundle-js-headers-response.msft.png)
+![The Lighthouse opportunity section showing the unused JS message](./images/unused-javascript-opportunity.png)
```

Comment:
```suggestion
![The Lighthouse opportunity section showing the unused JavaScript message](./images/unused-javascript-opportunity.png)
```
need to check rendered page.  maybe write:
showing the "Unused JavaScript" message